src/pages/dashboard/_MapContainer.js

```diff
@@ -136,6 +136,7 @@ const MapContainer = () => {
 
   return (
     <div className="state-map">
+      <h3>CUMULATIVE TESTS IN THE U.S.</h3>
```

Comment:
We have a different `h3` added here upstream at @julialedur's request. I think this is now incorrect.

https://github.com/COVID19Tracking/website/blob/a3c01a98245792e168daffc4c5ded5ac38b554a0/src/pages/dashboard/_MapContainer.js#L126-L128